--- fb303/FacebookService.py	2013-05-21 14:11:58.000000000 +0300
+++ fb303/FacebookService.py	2016-03-19 01:25:21.970917700 +0200
@@ -7,7 +7,7 @@
 #
 
 from thrift.Thrift import TType, TMessageType, TException, TApplicationException
-from ttypes import *
+from .ttypes import *
 from thrift.Thrift import TProcessor
 from thrift.transport import TTransport
 from thrift.protocol import TBinaryProtocol, TProtocol
