--- scribe/scribe.py	2013-05-21 14:12:01.000000000 +0300
+++ scribe/scribe.py	2016-04-29 19:52:14.822207521 +0300
@@ -8,7 +8,7 @@
 
 from thrift.Thrift import TType, TMessageType, TException, TApplicationException
 import fb303.FacebookService
-from ttypes import *
+from .ttypes import *
 from thrift.Thrift import TProcessor
 from thrift.transport import TTransport
 from thrift.protocol import TBinaryProtocol, TProtocol
